### DISCUSSION TITLE
Is there a community meeting to discuss the updates in the project?

### DISCUSSION THREAD

**Discussion by lakshya8066** [wolfi-dev/os#36111] [General]
Hi I am new to the community and like to get involved in the project. It would be interesting to discuss the updates of the project with the team. Is there a mailing list or a calender invite?

**Comment by krisharyan117** [comment]
Hey Lakshya, the community meeting have stopped and have mostly moved to async questions[ here](https://kubernetes.slack.com/archives/C03MASRQP4M) and through other channels. You can checkout past meeting recording on Josh's youtube channel: [link](https://www.youtube.com/@joshdolitsky2081)
